packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -342,6 +345,9 @@ export abstract class FluidDataStoreContext
 				.catch((error) => {});
 		}
 	}
+	public delete() {
```

Comment:
Can we get a comment here?  The concept of deleting a dataStoreContext is brand new.  Would be good to explain what it does/doesn't mean.

Bonus nit: Missing blank line above this